manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "vendor": "vtex",
   "name": "search-graphql",
-  "version": "0.23.1",
+  "version": "0.24.0-beta.0",
```

Comment:
remove this change

Reply:
This is removed by vtex release and our CI bot understands it. If I select minor, it will deploy a `v0.24.0`, if I select patch it will deploy a `v0.23.2`.